x-pack/plugins/threat_intelligence/public/components/indicator_field/indicator_field.tsx

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
I'm on a fence about having this component in the top-level `components`. It's true that it is stateless and also can be reused in multiple places in the codebase, but I have a feeling the more code we have the more confusion we'll have about what is a component, what is a container, and what should go in the modules and what on the top level. In Kibana a lot of features can appear in very different contexts, for example, Indicator Flyout can easily appear in a Case or in Timeline.
 Maybe having a generic `common` on the top level is not the best idea long term and we should look more into separating our code by "domains". Eg. have a structure smth like that:
```
/modules
  /indicators
  /empty_page
  /dates
  /kibana
  /security_solution
  /filters
```
Otherwise we might end up with a mess inside top level `/components`, '/containers` and `/hooks`
I don't have a very well picture in my head yet, added the topic for discussion in our tech planning. For now I'm fine with merging it as is or if you think it makes sense you can move indicator_field inside `modules/indicators`

Reply:
I'm happy to move it inside `indicators` as I agree with you, it doesn't feel like it belongs at the very top level.

As for the modules here, I will follow your lead. In Angular, modules correspond to features so for me it wouldn't make sense to have dates or security_solution as a module but I'm ok with this approach as well (although I won't make those drastic changes in this PR or it would change too much from what it is now)

Reply:
I moved the IndicatorField component under modules/indicators.

Happy to clean up the rest of the folder structure once we've discussed about it in our next tech planning!